web/src/pages/Route/components/DebugViews/AuthenticationView.tsx

```diff
@@ -108,7 +108,7 @@ const AuthenticationView: React.FC<RouteModule.DebugViewProps> = (props) => {
               onChange={(event) => {
                 const currentValue = event.target.value;
                 setAuthType(currentValue);
-                props.form.setFieldsValue({ authType: currentValue });
+                props.form.setFieldsValue({ autyType: currentValue });
```

Comment:
May I know why change this back?

Reply:
the whole feature has been remove in current version